android/src/org/ppsspp/ppsspp/NativeActivity.java

```diff
@@ -1409,12 +1409,6 @@ public boolean processCommand(String command, String params) {
 				// Only keep the screen bright ingame.
 				window.clearFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
 			}
-		} else if (command.equals("event")) {
```

Comment:
Why is this removed?

Reply:
this is called from __UsbCamShutdown now

Reply:
No not this Android mCameraHelper, as far as I can tell skimming the code...